### DISCUSSION TITLE
Colour command prompt.

### DISCUSSION THREAD

**Discussion by macerg** [termux/termux-packages#9320] [Q&A] [answered]
Please forgive me if asked before or not correct place for this.
How can I get a colour command line prompt in termux?
I've messed with .bashrc but just can't get it to work.

**Comment by ChiefMikeK** [comment]
Edit the colors.properties file
```
nano ~/.termux/colors.properties

       Here is mine 
﹩cat ~/.termux/colors.properties
# Scheme: modified bespin-dark
foreground=#efebe9
background=#28211c
cursor=#f8bbd0

color0=#28211c
color1=#cf6a4c
color2=#54be0d
color3=#f9ee98
color4=#5ea6ea
color5=#9b859d
color6=#afc4db
color7=#8a8986
color8=#666666
color9=#cf6a4c
color10=#54be0d
color11=#f9ee98
color12=#5ea6ea
color13=#9b859d
color14=#afc4db
color15=#baae9e

color16=#cf7d34
color17=#937121
color18=#36312e
color19=#5e5d5c
color20=#797977
color21=#9d9b97

```

**Reply by macerg** [reply]
Thanks for that,  ChiefMikeK.
I did that. In my system, it did absolutely nothing. Yes, I did start a new session.

**Reply by macerg** [reply]
Correction, I can see that the cursor 'block' has changed to a pale pink. That's ok but not what I had in mind. I'll post an example soon. Thanks.

**Comment by macerg** [comment]
I'd like this bit:-
richard@localhost:~

to be in green if possible, like it is on my Debian and Ubuntu systems.

**Comment by macerg** [comment, marked as answer]
Thanks, found the answer here:-
https://www.geeksforgeeks.org/how-to-customize-bash-colors-and-content-in-linux-terminal-prompt/
